p0.py

```diff
@@ -0,0 +1,96 @@
+#This is the first code I've ever written in python... maybe it works! 
```

Comment:
You should remove this p0.py (e.g. git rm p0.py), but keep the p0.py that is located in 0/rkm79791/ . That way p0.py only shows up where it should.